libs/hdf-converters/src/scoutsuite-mapper.ts

```diff
@@ -220,51 +219,73 @@ export class ScoutsuiteMapper extends BaseConverter {
             }
           }
         ],
-        depends: [],
         groups: [],
         status: 'loaded',
         controls: [
           {
             path: 'services',
             key: 'id',
-            id: {path: '[0]'},
-            title: {path: '[1].description'},
             tags: {
               nist: {path: '[0]', transformer: nistTag}
             },
-            impact: {
-              path: '[1].level',
-              transformer: impactMapping(IMPACT_MAPPING)
-            },
+            refs: [
+              {url: {path: '[1].references[0]'}},
+              {ref: {path: '[1].compliance', transformer: compliance}}
+            ],
+            source_location: {},
+            title: {path: '[1].description'},
+            id: {path: '[0]'},
             desc: {path: '[1].rationale'},
             descriptions: [
               {data: {path: '[1].remediation'}, label: 'fix'},
               {data: {path: '[1].service'}, label: 'service'},
               {data: {path: '[1].path'}, label: 'path'},
               {data: {path: '[1].id_suffix'}, label: 'id_suffix'}
             ],
-            refs: [
-              {url: {path: '[1].references[0]'}},
-              {ref: {path: '[1].compliance', transformer: compliance}}
-            ],
-            source_location: {},
-            code: '',
+            impact: {
+              path: '[1].level',
+              transformer: impactMapping(IMPACT_MAPPING)
+            },
+            code: {
+              transformer: (vulnerability: Record<string, unknown>): string =>
+                JSON.stringify(vulnerability, null, 2)
+            },
             results: [
               {
                 status: {path: '[1]', transformer: getStatus},
                 skip_message: {path: '[1]', transformer: checkSkip},
-                message: {path: '[1]', transformer: getMessage},
                 code_desc: {path: '[1].description'},
+                message: {path: '[1]', transformer: getMessage},
                 start_time: {path: '$.last_run.time'}
               }
             ]
           }
         ],
         sha256: ''
       }
-    ]
+    ],
+    passthrough: {
+      transformer: (data: Record<string, unknown>): Record<string, unknown> => {
+        const auxData = _.omit(data, [
+          'account_id',
+          'environment',
+          'partition',
+          'provider_code',
+          'provider_name',
+          'services'
+        ]);
+        if (auxData instanceof Object) {
```

Comment:
what is the purpose of this conditional?

Reply:
Type guard so the `auxData` variable's data type will be narrowed from `unknown` to `Object` so I can access attributes.

Reply:
It did when I wrote it, but I checked again and it doesn't seem to be an issue now. Removed the type guard and the mapper still works just fine.